Dockerfile.build

```diff
@@ -15,3 +15,5 @@ RUN apt-get update -qq \
    "deb [arch=amd64] https://download.docker.com/linux/$(. /etc/os-release; echo "$ID") $(lsb_release -cs) stable"
 RUN apt-get update -qq \
  && apt-get install -qq -y --force-yes docker-ce=17.12.0~ce-0~debian
+
+WORKDIR /src
```

Comment:
the default `WORKDIR` is `/go`. why change it?

Reply:
If you use the default, that's in the go path, making go modules not work. The path should be changed - `/src` is what I've tested successfully before - to allow for modules to properly be invoked.

Really frustrating, I know :(

Reply:
Ugh